### ISSUE TITLE
customTheme overflow

### ISSUE THREAD

**Issue by maxelcalamar** [evoluteur/colorpicker#43]
Color squares overflowing, when adding more than 9 colors to the custom theme

![image](https://user-images.githubusercontent.com/5317548/32514476-dab461d6-c3db-11e7-8072-1694ab7f2f39.png)

Code:
$('#customTheme').colorpicker({
		color: '#f44336',
		customTheme: ['#f44336','#ff9800','#ffc107','#4caf50','#00bcd4','#3f51b5','#9c27b0', 'white', 'black','#f44336','#ff9800','#ffc107','#4caf50','#00bcd4','#3f51b5','#9c27b0', 'white', 'black']
	});

**Comment by pitgas** [comment]
Hi same problem, is there a solution?
cu.

**Comment by rfree190** [comment]
I know the question is fairly old, but I was having the same issue.  Thought I'd post the solution I came up with in case someone else came looking too.

Looks like JS is creating a table and giving it a class of evo-palette.  I added this CSS and it seems to be working the way I want.  Hopefully this code will help someone else, too.

table.evo-palette td {
	width: 22px;
	height: 22px;
	display: block;
	float: left;
	margin: 2px;
}
